packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import * as _ from 'underscore';
+
+import {IButtonProps} from '../button/Button';
+import {InputConnected} from '../input/InputConnected';
+import {multilineBoxWithRemoveButton} from '../multilineBox/hoc/MultilineBoxWithRemoveButton';
+import {IMultilineParentProps, IMultilineSingleBoxProps, MultilineBox} from '../multilineBox/MultilineBox';
+
+const MultilineBoxWithRemoveButton = _.compose(
+    multilineBoxWithRemoveButton({
+        containerNode: (
+            child: React.ReactNode,
+            getRemoveButton: (props?: Partial<IButtonProps>) => React.ReactNode,
+            data: Array<IMultilineSingleBoxProps<string>>,
+            index: number
+        ) => (
+            <div key={`${data[index].id}`} className="flex">
+                <div className="flex-auto">{child}</div>
+                {getRemoveButton({
+                    classes: ['mod-no-border', 'bg-transparent', 'align-end'],
+                })}
+            </div>
+        ),
+    })
+)(MultilineBox);
+
+export interface MultiValuesInputProps {
+    id: string;
+    data: string[];
+}
+
```

Comment:
I'm not sure if it's out of scope. `RemoveButton` appears if the input is not the last, but do we need an `AddButton` for the last one? 
![image](https://user-images.githubusercontent.com/52677246/76108873-619cd000-5fa9-11ea-9d75-4767c55be22a.png)

Reply:
The input is added as soon as the value is not empty so it's ok to not have the add button